SEED-platform/seed
# Taxlot geocoding update

#### Any background context you want to provide?
There were multiple issues with Tax Lot geocoding - specifically involving ULID.  These essentially came from the previous (and now invalid) assumption that Tax Lots would not be eligible for manual geocoding via provided long/lat or ULID.

#### What's this PR do?
Addresses the issues listed below.
- Geocoding logic update to enable manual geocoding for tax lots
- Fix import dedup fail with pre_save to sync taxlot lat and long
- Import feedback provided for manual geocoding of taxlots via ULID (update relevant view, controller, and partial)

#### How should this be manually tested?
1. Import TaxLots with ULID, checking to see that each record with a valid ULID is accounted for in the import results feedback.
2. Import that same file again and be sure that everything was flagged as a duplicate.
3. Check that any/all of the entries were geocoded manually - Geocoding confidence should be set to "Manually geocoded (N/A)".

#### What are the relevant tickets?
- #1950 
- #1944 
- #1942 

#### Screenshots (if appropriate)
